TEAM_STRUCTURES.md

```diff
@@ -131,38 +135,26 @@ The QA, Testing and Dev Team Enablement Working Group focuses on building develo
 - Define process and help achieve high quality sustainable open source modules.
 - Automating tasks for things such as: release cycle, ensuring code quality, activity reports.
 
-### Community, Events & Evangelism
+### Community
 
 > This Working Group hasn't been formed yet.
```

Comment:
Would you say that with this PR the Working Group is formed?

Reply:
Yes :)